src/RenderTarget.js

```diff
@@ -30,8 +30,23 @@ class RenderTarget {
   constructor(view) {
     this.view = view;
     this.eventEmitter = new EventEmitter();
-    this.ctx = polyfillCanvasCtx(wx.createCanvasContext('writer-canvas', view));
-    this.canvas = this.view.selectComponent('#writer-canvas');
+    //this.ctx = polyfillCanvasCtx(wx.createCanvasContext('writer-canvas', view));
+    //this.canvas = this.view.selectComponent('#writer-canvas');
+    
+    // repalce `createCanvasContext`(2.9.0 deprecated) with `createSelectorQuery`
```

Comment:
nit: replace

Reply:
I have deleted the `replace` commented line.

Reply:
should I also delete these lines:
```
    //this.ctx = polyfillCanvasCtx(wx.createCanvasContext('writer-canvas', view));
    //this.canvas = this.view.selectComponent('#writer-canvas');
```